frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -76,7 +77,7 @@ export function EmbeddingSdkSettings({
   }
 
   function handleToggleEmbeddingSdk(event: ChangeEvent<HTMLInputElement>) {
-    updateSetting({ key: "enable-embedding-sdk" }, event.target.checked);
+    updateSetting({ key: "enable-embedding-sdk" }, event.currentTarget.checked);
```

Comment:
https://v6.mantine.dev/core/switch/#controlled - we should be using `currentTarget`